src/plugins/timeline/TimelineViewLayout.vue

```diff
@@ -105,7 +120,7 @@ export default {
     addItem(domainObject) {
       let type = this.openmct.types.get(domainObject.type) || unknownObjectType;
       let keyString = this.openmct.objects.makeKeyString(domainObject.identifier);
-      let objectPath = [domainObject].concat(this.objectPath.slice());
+      let objectPath = [domainObject].concat(this.path.slice());
```

Comment:
we can do something like this to keep them distinct
```suggestion
      let newObjectPath = [domainObject].concat(this.objectPath.slice());
```